src/api/init.ts

```diff
@@ -128,6 +128,16 @@ export async function startApiServer(opts: {
     });
     app.use(promMiddleware);
   }
+  // adding stacks-api-version in header
+  app.use((_, res, next) => {
+    try {
+      const [branch, commit, tag] = fs.readFileSync('.git-info', 'utf-8').split('\n');
+      res.setHeader('X-Stacks-API-Version', `stacks-blockchain-api ${tag} (${branch}:${commit})`);
```

Comment:
This header needs added to CORs `Access-Control-Expose-Headers` allowlist if we want web browser js to be able to access it.